### DISCUSSION TITLE
Changing date placeholder

### DISCUSSION THREAD

**Discussion by stighagen** [argenos/zotero-mdnotes#117] [Q&A]
Hi there,

Thanks for a great plug-in! I'm customizing my own templates and would like to remove the wiki-link style from the date placeholder but keep the links elsewhere. I can't find the date placeholder in the Zotero Config Editor. Any ideas?

My idea is to use the placeholder to populate YAML meta on top of the document, and I need a pure literature date.

**Comment by argenos** [comment]
You'd have to add the custom formatting rule (when there are no preferences defined, fields just use the default: https://argentinaos.com/zotero-mdnotes/docs/next/advanced/formatting#placeholder-format)

**Reply by stighagen** [reply]
Would this work:
`extensions.mdnotes.placeholder.date {"content":"{{field_contents}}"}`

Is it possible to add a second instance of date without messing with the first? If you don't mind please give an example, I'm a novice, and don't know programming :-)

Thank you!

**Reply by stighagen** [reply]
And since I'm adding a new instance, what exactly happens with the existing one that I can't search and modify?

**Reply by stighagen** [reply]
Ahh… understand now. Thanks.
